cluster/src/main/java/com/hartwig/pipeline/tools/Versions.java

```diff
@@ -30,6 +30,7 @@
     String CHORD = "60.02_1.03";
     String SAMTOOLS = "1.9";
     String BCL2FASTQ = "2.20.0.422";
+    String BAMCOMP = "0.1";
```

Comment:
0.1 implies its not ready for production, would suggest we use 1.0 once its validated (which I think it sort of is by Korneel)